README.md

```diff
@@ -0,0 +1,53 @@
+### Example of integrating with inner blocks in the WooCommerce Blocks Checkout
+
+This repository contains some example code showing how an extension can register an inner block for use in the Checkout
+Block offered by WooCommerce Blocks.
+
+To integrate a front-end block with WooCommerce Blocks that extends the API, work is required in two areas:
+Server-side code that extends the `StoreApi` and front-end code that renders and controls your block.
```

Comment:
```suggestion

- server-side code that extends the `StoreApi` 
- front-end code that renders and controls your inner block
```

Reply:
I'd also add some links to the sections below here.

Reply:
> I'd also add some links to the sections below here.

Could you clarify what you mean please? Do you mean links to the PHP and JS sections in this same document?

Reply:
yes, exactly